recipes/hpack/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "hpack" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "8eec9c1f4bfae3408a3f30500261f7e6a65912dc138526ea054f9ad98892e9d2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
This [should be `noarch: python`](https://conda-forge.org/docs/meta.html#building-noarch-packages), I think.